src/ucp/tag/tag_send.c

```diff
@@ -49,8 +49,9 @@ static UCS_F_ALWAYS_INLINE ucs_status_ptr_t
 ucp_tag_send_req(ucp_request_t *req, size_t dt_count,
                  const ucp_ep_msg_config_t* msg_config,
                  size_t rndv_rma_thresh, size_t rndv_am_thresh,
-                 ucp_send_callback_t cb, const ucp_request_send_proto_t *proto,
-                 int enable_zcopy)
+                 ucp_send_nbx_callback_t cb, void *user_data,
+                 const ucp_request_send_proto_t *proto,
+                 int enable_zcopy, int enable_cb)
```

Comment:
maybe check for `cb == NULL` instead of enable_cb?

Reply:
ok